packages/storage-plus/src/keys.rs

```diff
@@ -19,6 +19,16 @@ pub trait PrimaryKey<'a>: Clone {
     }
 }
 
+// Empty / no primary key
+impl<'a> PrimaryKey<'a> for () {
```

Comment:
Now I get it...

`Item<T>` is just another name for `Map<(), T>` (but with more clear methods).
Now I see where re-use comes in. Nice idea.